Script.py

```
@@ -140,9 +139,9 @@ class script(object):
     
     PLAN_TEXT = """<b>ᴡᴇ ᴀʀᴇ ᴘʀᴏᴠɪᴅɪɴɢ ᴘʀᴇᴍɪᴜᴍ ᴀᴛ ᴛʜᴇ ʟᴏᴡᴇsᴛ ᴘʀɪᴄᴇs:
     
-1 ʀᴜᴘᴇᴇ ᴘᴇʀ ᴅᴀʏ 👻
-29 ʀᴜᴘᴇᴇs ғᴏʀ ᴏɴᴇ ᴍᴏɴᴛʜ 😚
-55 ʀᴜᴘᴇᴇs ғᴏʀ ᴛᴡᴏ ᴍᴏɴᴛʜs 😗
+𝟺𝟿 ʀᴜᴘᴇᴇ ᴘᴇʀ ᴍᴏɴᴛʜ 👻
```

Ruff (0.13.2)

String contains ambiguous `𝟺` (MATHEMATICAL MONOSPACE DIGIT FOUR). Did you mean `4` (DIGIT FOUR)?

(RUF001)